.gitignore

```diff
@@ -5,6 +5,7 @@ pkg/
 *.swp
 *~
 .vscode/
+.vs/
 .vs/slnx.sqlite
```

Comment:
I think you can remove these 2 lines now?
also you might need to do `git rm -rf .vs/`

Reply:
My previous commit should have removed the only reference .vs in the codebase `find . -name ".vs"` returns nothing